.mailmap

```diff
@@ -51,3 +51,4 @@ Thomas Gazagnaire <thomas.gazagnaire@docker.com> <thomas@gazagnaire.com>
 Thomas Gazagnaire <thomas.gazagnaire@docker.com> <thomas@gazagnaire.org>
 Vincent Demeester <Vincent.Demeester@docker.com> <vincent@sbr.pm>
 Vincent Demeester <Vincent.Demeester@docker.com> <vdemeester@docker.com>
+Tiejun Chen <tiejun.china@gmail.com> <tiejunc@vmware.com>
```

Comment:
Could you keep this list sorted alphabetically?

Reply:
@thaJeztah Got it.

Reply:
This request is still not addressed. Please keep the list sorted